Extension/src/LanguageServer/client.ts

```diff
@@ -2499,18 +2499,9 @@ export class DefaultClient implements Client {
             const status: IntelliSenseStatus = { status: Status.TagParsingBegun };
             testHook.updateStatus(status);
         } else if (message.endsWith("Initializing")) {
-            if (ui.isNewUI) {
-                this.model.isInitializingWorkspace.Value = true;
-            } else {
-                this.model.isParsingWorkspace.Value = true;
-            }
+            this.model.isInitializingWorkspace.Value = true;
         } else if (message.endsWith("Indexing")) {
-            if (ui.isNewUI) {
-                this.model.isIndexingWorkspace.Value = true;
-                this.model.isInitializingWorkspace.Value = false;
-            } else {
-                this.model.isParsingWorkspace.Value = true;
-            }
+            this.model.isIndexingWorkspace.Value = true;
```

Comment:
The original code the these properties set if the new UI was being used:

```              
this.model.isIndexingWorkspace.Value = true;
this.model.isInitializingWorkspace.Value = false;
```

It looks like `this.model.isInitializingWorkspace.Value = false;` got deleted or was it no longer needed?

Reply:
Thanks for catching that!